api/mraa/gpio.h

```diff
@@ -220,6 +220,14 @@ int mraa_gpio_get_pin(mraa_gpio_context dev);
  */
 int mraa_gpio_get_pin_raw(mraa_gpio_context dev);
 
+/**
+ * Get the SYSFS pin path, invalid will return NULL
+ *
+ * @param dev The Gpio context
+ * @return Gpio pin sysfs path
```

Comment:
but it already has "invalid will return NULL" in the same way other functions mention.